### PR TITLE
pc - clarify mongo db instructions

### PR DESCRIPTION
**Pull request by pconrad**
In this PR, we add an additional image to help new users
navigate the creation of their first database.    The dialog
is different from them; there is no  button.

### CODE REVIEW

File: docs/mongo-setup.md [hunk 1]
```diff
@@ -7,11 +7,20 @@ add yourself as an admin user.
 
    ![collections](./images/collections.png)
 
-2. Click "+ Create Database" to create a database. Make sure you name it "database" and create a collection called "users".
+2 If you see "+ Create Database", click that to create a database.
```
Comment (btk5h) [inline docs/mongo-setup.md:10]: I think this is missing the period in the 2. to make this a list

Comment (pconrad) [reply docs/mongo-setup.md]: Good catch.  Indentation was incorrect too.    Fixed now, I think.

**Comment by pconrad** [comment]
Also, reduced size of  image on step 2 to be of a better scale for the page.

**Comment by pconrad** [comment]
Requested changes made.. merging...

**Pull request merged**